.github/workflows/cancel.yml

```
@@ -0,0 +1,15 @@
+name: Cancel
+on: [pull_request, pull_request_target]
+jobs:
+  cancel:
+    name: 'Cancel Previous Runs Test'
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+    steps:
+      - name: Get all workflow ids and set to env variable
+        run: echo ::set-env name=WORKFLOW_IDS_TO_CANCEL::$(curl https://api.github.com/repos/${GITHUB_REPOSITORY}/actions/workflows -s | jq -r '.workflows | map(.id|tostring) | join(",")')
+
+      - uses: styfle/cancel-workflow-action@0.4.0
+        with:
+          workflow_id: ${{ env.WORKFLOW_IDS_TO_CANCEL }}
+          access_token: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
          access_token: ${{ secrets.MLPACK_GITHUB_ACTIONS }}
```

Token created, maybe that works?

---

This is showing some error.

---

Have you install the app.

---

I guess this is passing nothing.